packages/libs/sdk/package.json

```diff
@@ -7,14 +7,14 @@
   },
   "license": "MIT",
   "version": "1.0.0-alpha.0",
-  "main": "./src/index.js",
-  "module": "./src/index.esm.js",
-  "types": "./src/index.d.ts",
+  "type": "module",
```

Comment:
I updated this because I'm trying to optimize for esmodules, so figured it would be helpful if this is the default locally. I don't think it matters which we use locally because of the individual `exports` below and the script to add a package.json for each bundle's subdirectory (`tools/scripts/sdk_package_json.mjs`)